src/databricks/labs/ucx/framework/tasks.py

```diff
@@ -242,6 +242,11 @@ def run_task(
     workflow_run_id = args.get("parent_run_id", "unknown_run_id")
     job_id = args.get("job_id", "unknown_job_id")
 
+    # Workaround to pass workflow ids to `parse_logs` tasks
```

Comment:
we don't need it anymore, please rebase on main

Reply:
outdated